### PR TITLE
Add run-bash-script check

### PR DESCRIPTION
**Pull request by zakhark**

### CODE REVIEW

File: redalert/checks/run_bash_script.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import tempfile
+
+from .exc import CheckFailure
+
+class RunBashScript:
+    """A check for running a bash script and optionally checks output.
+
+    Takes the following arguments:
+    source (required): The source code of the script.
+    output: expected output string.
```
Comment (chasinglogic) [inline redalert/checks/run_bash_script.py]: nit: I would write this as "string which the output of the script will be compared to determine a successful run. If omitted only checks that the script exits with returncode 0"

The first sentence can stay the same as yours but I think we should mention the behavior when output is ommited

File: redalert/checks/run_bash_script.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    interpreter: path to bash. Default is '/bin/bash'.
```
Comment (chasinglogic) [inline redalert/checks/run_bash_script.py]: nit: I would write this as "path to shell used to run the script. Default is '/bin/bash'"

File: tests/test_run_bash_script.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+def test_run_bash_script_raises_on_mismatch_output():
+    bs = RunBashScript(source='echo "111"', output="123")
+    with pytest.raises(CheckFailure):
+        bs.check()
+
+def test_run_bash_script_doesnt_raise_on_good_interpreter():
+    bs = RunBashScript(source='echo "111"', interpreter='/bin/sh')
+    bs.check()
+
+def test_run_bash_script_raises_on_bad_interpreter():
```
Comment (chasinglogic) [inline tests/test_run_bash_script.py]: Can we add a test that a bad command also causes a check failure?

Something like `source='ls -l /not_a_directory'` I want to replace command-group-all and command test types from Greenbay with run-bash-script so I want to have a test to verify the behavior is what we expect and that we shouldn't try to add extra options to bash when run it.

**Comment by zakhark** [comment]
I've changed this to run-script but kept an alias for run-bash-script to make the transition easier.

Addressed comments as well

**Pull request merged**